library/core/class.oauth2.php

```diff
@@ -59,6 +62,18 @@ class Gdn_OAuth2 extends Gdn_Plugin implements \Vanilla\InjectableInterface {
     /** @var  @var string optional set the settings view */
     protected $settingsView;
 
+    /**
+     * Gets the $authenticaitonScheme variable
+     *
+     * @return string
+     */
+    protected function getAuthenticationScheme(): string {
+        if (!$this->authenticationScheme) {
+            return '';
+        }
+        return $this->authenticationScheme;
+    }
+
```

Comment:
After testing this with other plugins that extend this class already, this should be:
```
protected function getAuthenticationScheme(): string {
    return $this->authenticationScheme ?: $this->providerKey;
}
```
because that is the logic we have been using.